docker-compose.yml

```diff
@@ -21,7 +21,7 @@ services:
     environment:
       - NETWORK=${NETWORK}
     volumes:
-      - cardano-node:/data
+      - node-ipc:/ipc
```

Comment:
`/ipc` ? is it a typo or is the volume mounted on `/ipc` in the cardano-node image?

Reply:
Not a typo as of https://github.com/input-output-hk/cardano-node/blob/master/nix/docker.nix#L94

Reply:
Done